src/main/java/pl/codilingus/library/validation/OrderValidator.java

```diff
@@ -0,0 +1,45 @@
+package pl.codilingus.library.validation;
+import org.springframework.beans.factory.annotation.Autowired;
+import pl.codilingus.library.Book;
+import pl.codilingus.library.Order;
+import pl.codilingus.library.controllers.OrderController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OrderValidator {
+
+    @Autowired
+    private OrderController orderController;
```

Comment:
We should avoid injecting Controllers anywhere, it'd be better to replace it with OrderRepository here if possible :)